apps/dash-lunar-lander/app.py

```diff
@@ -0,0 +1,438 @@
+import dash
+import dash_core_components as dcc
+import dash_html_components as html
+from dash.dependencies import Input, Output
+import numpy as np
+import casadi
+from casadi import SX, DM
+from math import cos, sin
+import plotly.graph_objects as go
+
+app = dash.Dash(__name__)
+server = app.server
+
+app.layout = html.Div(
+    [
+        html.Div(
+            children=[
+                html.Img(src=app.get_asset_url("dash-logo.png"), id="logo"),
+                html.H4("Dash Lunar Lander"),
+                html.A(
+                    id="github-link",
```

Comment:
You could also change it to a button-like html.A https://github.com/plotly/dash-bio/blob/4d7bd372927a1c2f4c333506130192c73ff337af/tests/dashbio_demos/common/layout_helper.py#L76-L87 . Whatever looks good (up to your decision) 😸